sdk/response.js

```diff
@@ -84,10 +84,7 @@ class Response extends Api {
    * @return {boolean}
    */
   hasReturn() {
-    return this
-      .getServiceSchema(this._name, this._version)
-      .getActionSchema(this._actionName)
-      .hasReturn();
+    return this.return !== undefined;
```

Comment:
The *userland* expectation is to receive an exception, so that cannot be changed. You would have to catch and process it accordingly.